.gitignore

```diff
@@ -56,16 +56,23 @@ __pycache__/
 /External/catch/catch/
 /External/CoordGen/coordgen/
 /External/CoordGen/maeparser/
-/External/FreeSASA/freesasa-2.0.1/
+/External/FreeSASA/freesasa-2.0.*/
```

Comment:
Please revert .gitignore

Reply:
I am OK to revert, but the files generated during the build process are not enough with the current ".gitignore" and current RDKit uses `freesasa-2.0.3` not `freesasa-2.0.1`. 
https://github.com/rdkit/rdkit/blob/44cf2c9e5705111214b4ca4933604ea5bc7ce058/External/FreeSASA/CMakeLists.txt#L15

Reply:
That makes sense.  You can just do freesasa* then

Reply:
Why don't we make this a separate issue to update .gitignore so we don't mix the two problems.  This will allow us to also see if we need to add anything else.

Reply:
OK. I will make a revert commit.